src/app/store/effects/pokemon.effects.ts

```diff
@@ -0,0 +1,155 @@
+import { Injectable } from '@angular/core';
+import { MatDialog } from '@angular/material/dialog';
+import { Actions, createEffect, ofType } from '@ngrx/effects';
+import { select, Store } from '@ngrx/store';
+import { first, tap } from 'rxjs/operators';
+import { CurrentPokemonService } from 'src/app/services/current-pokemon/current-pokemon.service';
+import { DialogPokemonComparationComponent } from 'src/app/material/dialog-pokemon-comparison/dialog-pokemon-comparation.component';
+import { DialogPokemonInfoComponent } from 'src/app/material/dialog-pokemon-info/dialog-pokemon-info.component';
+import { FullAlertComponent } from 'src/app/material/full-alert/full-alert.component';
+import { PokemonListService } from 'src/app/services/pokemon-list/pokemon-list.service';
+import * as PokemonActions from '../actions/pokemon.actions'
+import { setPokemonsToShow } from '../actions/pokemon.actions';
+import { selectFromStore } from '../selectors/pokemon.selectors';
+
+
+@Injectable()
+export class PokemonEffects {
+
+  loadPokemons$ = createEffect(() =>
+    this.actions$.pipe(
+      ofType(PokemonActions.getPokemonFromApi),
+      tap(
+        () => {
+          this.listService.getAllPokemons();
+        })
+    ), { dispatch: false });
```

Comment:
effects are meat to be used to dispatch effects, the dispatch:false configuration should be avoided